guide/additional-info/miscellaneous-examples.md

```diff
@@ -0,0 +1,161 @@
+# Miscellaneous examples
+
+<p class="tip">For more, even shorter code examples, check out [the common questions page](/creating-your-bot/common-questions)!</p>
+
+### Play music from YouTube
+
+Here's a short example of a command that connects to a voice channel, plays a song, and exits when it's done. You'll need to install the `ytdl-code` package in order to run this.
+
+```
+npm install --save ytdl-core
+```
+
+```js
+const Discord = require('discord.js');
+const ytdl = require('ytdl-core');
+
+const client = new Discord.Client();
+
+client.on('message', message => {
+	if (message.content.startsWith('!play')) {
+		if (message.channel.type !== 'text') return;
+
+		const { voiceChannel } = message.member;
+
+		if (!voiceChannel) {
+			return message.reply('please join a voice channel first!');
+		}
+
+		voiceChannel.join().then(connection => {
+			const stream = ytdl('https://www.youtube.com/watch?v=D57Y1PruTlw', { filter: 'audioonly' });
+			const dispatcher = connection.playStream(stream);
+
+			dispatcher.on('end', () => voiceChannel.leave());
+		});
+	}
+});
+
+client.login('your-token-goes-here');
+```
+
+### Catching UnhandledPromiseRejectionWarnings
+
+If you've ever seen something labled `UnhandledPromiseRejectionWarnings` in your console, it can sometimes be a bit vague as to why it occured. In addition to that, not handling promise rejections is currently deprecated and will eventually be removed. Once that happens, any unhandled promise rejections will close your app with a non-zero exit code.
```

Comment:
`it can sometimes be a bit vague as to why it occured` -> `the reason of occurrence can be vague` or something like this

Reply:
Also, I think it is just `unhandledPromiseRejection` *warnings*, like, you listen for that event. I think that changes things, not sure.